### DISCUSSION TITLE
Error while setting up the project // asset-mapper wrong version

### DISCUSSION THREAD

**Discussion by ChrKn** [symfony/symfony#54168] [Symfony Book: The Fast Track] [answered]
Hello, I'm completely new to Symfony and currently trying to work through the book to get the basics, but ran into a dead-end quite quickly.

According to `symfony book:check-requirements`  I'm good to start. When I run `symfony new guestbook --version=6.2 --php=8.1 --webapp --docker --cloud` to initialize the project I run into "_Root composer.json requires symfony/asset-mapper 6.2.*, found symfony/asset-mapper[v6.3.0-BETA1, ..., 6.4.x-dev, v7.0.0-BETA1, ..., 7.1.x-dev] but it does not match the constraint._" 

Now my assumption is, that since 6.2 is no longer maintained, I would need to downgrade my version of Symfony in order to get the version it needs, but have no clue how or if that even is the issue. In a regular project I would simply change my composer.json to fix this, but since everything is loaded from git I can't really fiddle with that. 

Pointers would be very much appreciated. 

Windows 10; Symfony CLI (5.8.11) installed as recommended via scoop.

**Comment by smnandre** [comment, marked as answer]
Well it seems related to the `--cloud` option.

This is working as expected

```
symfony new guestbook --version=6.2 --php=8.1 --webapp
```

But this generates the error you mentionned

```
symfony new guestbook --version=6.2 --php=8.1 --webapp --cloud
```

Any particular reason you want to use 6.2 ?

**Reply by ChrKn** [reply]
None whatsoever. Just following the guide and obviously not thinking for myself. I will give it a go with the 6 LTS version once I'm home again.

Yeah. So with 6.4 it runs through. 

`symfony new guestbook --version=6.4 --php=8.1 --webapp --docker --cloud`
